src/SFA.DAS.AANHub.Application/Apprentices/Commands/PatchApprenticeMember/PatchApprenticeMemberCommand.cs

```diff
@@ -0,0 +1,36 @@
+﻿using MediatR;
+using Microsoft.AspNetCore.JsonPatch;
+using SFA.DAS.AANHub.Application.Common.Validators.RequestedByMemberId;
+using SFA.DAS.AANHub.Application.Mediatr.Responses;
+using SFA.DAS.AANHub.Domain.Entities;
+
+namespace SFA.DAS.AANHub.Application.Apprentices.Commands.PatchApprenticeMember
+{
+    public class PatchApprenticeMemberCommand : IRequest<ValidatedResponse<PatchApprenticeMemberCommandResponse>>, IRequestedByMemberId
+    {
+        private const string Replace = "replace";
```

Comment:
This constant needs to be in a common place

Reply:
Or substitute with `[nameof(OperationType.Replace)](https://learn.microsoft.com/en-us/dotnet/api/microsoft.aspnetcore.jsonpatch.jsonpatchdocument.replace?view=aspnetcore-6.0)` is a possibility. I am already using it in an upcoming commit

Reply:
Replace has been replaced with `nameof(OperationType.Replace)` in all the relevant classes